.github/workflows/lint-test.yml

```diff
@@ -9,54 +9,29 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@v2
 
-      - name: Lint Code Base
-        uses: github/super-linter@v4
-        env:
-          DEFAULT_BRANCH: main
-          FILTER_REGEX_EXCLUDE: .*(README\.md|Chart\.yaml|NOTES.txt).*
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          MULTI_STATUS: false
-          VALIDATE_ALL_CODEBASE: false
-          VALIDATE_KUBERNETES_KUBEVAL: false
-          VALIDATE_YAML: false
+      - name: Fetch history
+        run: git fetch --prune --unshallow
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.1.0
+
+      - name: Run chart-testing (lint)
+        run: ct lint --config .github/ct-lint.yaml
 
   test:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
 
-      - name: Set up Helm
-        uses: azure/setup-helm@v1
-        with:
-          version: v3.4.2
-
-      - name: Add dependency chart repos
-        run: |
-          helm repo add fluidtruck https://fluidtruck.github.io/helm-charts
-          helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-          helm repo add grafana https://grafana.github.io/helm-charts
-          helm repo add stable https://charts.helm.sh/stable
+      - name: Fetch history
+        run: git fetch --prune --unshallow
 
       - name: Set up chart-testing
         uses: helm/chart-testing-action@v2.1.0
 
-      - name: Run chart-testing (lint)
-        run: ct lint --config .github/ct-lint.yaml
-
-      - name: Run chart-testing (list-changed)
-        id: list-changed
-        run: |
-          changed=$(ct list-changed --config .github/ct-list-changed.yaml)
-          if [[ -n "$changed" ]]; then
-            echo "::set-output name=changed::true"
-          fi
-
       - name: Create kind cluster
         uses: helm/kind-action@v1.2.0
-        if: steps.list-changed.outputs.changed == 'true'
```

Comment:
Bye bye only being able to change one chart at a time.

Reply:
Wait, sorry if I'm slow, but why are we dropping this?

Reply:
@GrantSheehan As our chart repo grows, we don't want to sit here and wait 20 minutes to apply a fix to one chart, then open another PR and repeat the process for another chart. This will allow us to treat this as a monorepo and apply fixes to multiple charts at once and release them all.